recipes/msaboot/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: msaboot
+  version: "0.1.0"
+
+source:
+  url: https://github.com/phac-nml/msaboot/archive/v0.1.0.tar.gz
+  sha256: 41a34899b4c768abec4bf4e15d5a0a08451208fab9403b00454f1b7203971fa6
+
+build:
+  number: 0
+  skip: True # [win]
```

Comment:
You can remove that `skip` line since there are no Windows builds on Bioconda anyway.